src/Illuminate/Support/Traits/EnumeratesValues.php

```diff
@@ -402,7 +402,7 @@ public function sum($callback = null)
     /**
      * Apply the callback if the value is truthy.
      *
-     * @param  bool  $value
+     * @param  mixed  $value
```

Comment:
Maybe `bool|mixed` would be better.

Reply:
But mixed already indicates bool?

Reply:
That's true, I was just thinking what would be the best way to indicate that this is the value for the truth test. Maybe I just overthinking things.